site/content/entry/citra-progress-report-2017-q4.md

```diff
@@ -140,18 +140,48 @@ time it takes for the service that scheduled it would run.
 Although this doesn't fix any reported bugs, it does make the timing emulation
 of Citra much more accurate.
 
-<!--
-
-TODO: Research
-
 ## [citra-qt : Fix a bug in our fullscreen implementation](https://github.com/citra-emu/citra/pull/3159) by [FearlessTobi](https://github.com/FearlessTobi)
 
-TODO: Write
+In Citra's GUI, a strange bug exists, where if you have fullscreen enabled, and
+you start a game while the window is maximized, the window instead unmaximizes
+before starting the game. This was due to Citra not keeping track of the window's
+state (be it normal, maximized, minimized, or fullscreen), and instead simply
+defaulting to normal. [FearlessTobi](https://github.com/FearlessTobi) found,
+[reported](https://github.com/citra-emu/citra/pull/3001#issuecomment-338401048),
+and fixed the bug. Citra now keeps track of the window's state, position, and
+size, every time it goes fullscreen and restores them every time it leaves
+fullscreen.
 
 ## [Allow input configuration with SDL joysticks](https://github.com/citra-emu/citra/pull/3116) by [muemart](https://github.com/muemart)
+
+In the input configuration menu, there has been the ability to change the key
+bindings on the keyboard for years now. Unfortunately, this menu allowed rebinding
+to keyboard keys only, not gamepads or any other input device one could use. This
+led to guides such as [this one](https://community.citra-emu.org/t/temporary-controller-configurations-for-citra/1061)
+being written, to help users manually change the configuration files if they
+wanted to use gamepads with Citra.
+
+This was extremely inconvenient and user-unfriendly to most, and so [muemart](https://github.com/muemart)
+took it upon themselves to finally add support for configuring gamepads within the
+configuration menu. Now, it's as simple as clicking a button, and pressing the
+corresponsing button on your controller, to set it up.
+
 ## [shader_jit_x64_compiler: Remove ABI overhead of LG2 and EX2](https://github.com/citra-emu/citra/pull/3145) by [MerryMage](https://github.com/MerryMage)
 
--->
+The shader JIT is a subsystem in dynarmic (Citra's ARM to x86 recompiler) is the
```

Comment:
the shader jit has nothing to do with dynarmic. Its entirely its own thing in video_core